db/db.js

```diff
@@ -46,8 +46,139 @@ const deleteExample = (request, response) => {
         response.status(200).send(`Example deleted with ID: ${id}`);
     });
 };
+//Get all menu items
+function getMenuItems(callback) {
+    pool.query("SELECT * FROM MenuItems", (error, results) => {
+        if (error) {
+            throw error;
+        }
+        callback(results.rows);
```

Comment:
To keep it consistent I think we want to do an else here
```suggestion
        } else {
        callback(results.rows);
        }
```